appveyor.yml

```diff
@@ -0,0 +1,40 @@
+version: "master-{build}"
+
+os: Windows Server 2012 R2
+platform:
+  - x64
+
+environment:
+  winrm_user: test_user
+  winrm_pass: Pass@word1
+  winrm_port: 5985
+  KITCHEN_YAML: c:\projects\windows-security-policy\.kitchen.appveyor.yml
+  SSL_CERT_FILE: c:\projects\windows-security-policy\certs.pem
+
+  matrix:
+    - ruby_version: "22"
+
+clone_folder: c:\projects\windows-security-policy
+clone_depth: 1
+branches:
+  only:
+    - master
+
+install:
+  - systeminfo
+  - winrm quickconfig -q
+  - ps: net user /add $env:winrm_user $env:winrm_pass
+  - ps: net localgroup administrators $env:winrm_user /add
+  - ps: $env:PATH="C:\Ruby$env:ruby_version\bin;$env:PATH"
+  - ps: Write-Host $env:PATH
+  - ps: ruby --version
+  - ps: gem --version
+  - ps: gem install bundler --quiet --no-ri --no-rdoc
+  - ps: Invoke-WebRequest -Uri http://curl.haxx.se/ca/cacert.pem -OutFile c:\projects\windows-security-policy\certs.pem
+  - ps: bundler --version
```

Comment:
You could either install ChefDK here:
```
. { iwr -useb https://omnitruck.chef.io/install.ps1 } | iex; install -project chefdk.
```
or add a gem file like https://github.com/dev-sec/chef-windows-hardening/blob/master/Gemfile